generate/generate.go

```diff
@@ -362,11 +434,16 @@ func (g *Generator) translateDecl(decl ast.Decl) error {
 			}
 			g.pfile.Service = append(g.pfile.Service, srv)
 		case *ast.Ident:
+			// TODO(vishen): Check to see if the ident is a known file
+			// option and add if to  `p.file.Options`.
```

Comment:
please no backticks in comments - they're not markdown.

Reply:
Fixed.